src/cljs/villagebookUI/components/sidebar.cljs

```diff
@@ -13,12 +13,17 @@
          [:p.sidebar-logo "villagebook"]
          [:div.divider]]
         [:div.sidebar-section
-         [:p.sidebar-section-head "Organisations"]
+         [:p.sidebar-section-head "Your organisations"]
          [:ul.sidebar-section-list
-          [:li.item
-           [:a.sidebar-link [utils/patch "#5fcc5f"] "Org 1"]]
-          [:li.item
-           [:a.sidebar-link [utils/patch "#ff8383"] "Org 2"]]
+          (doall
```

Comment:
This is needed since `for` generates a lazy sequence. Unless the sequence is expanded in place, reagent can't know if an atom is being de-refed in the sequence and can't make re-renders based on it (`org-store/get-selected` in this case ). [Relevant thread](https://github.com/reagent-project/reagent/issues/18)